astropy/io/ascii/ui.py

```diff
@@ -336,6 +336,9 @@ def write(table, output=None,  format=None, Writer=None, fast_writer=True, **kwa
 
     table = Table(table, names=kwargs.get('names'))
 
+    if table.has_mixin_columns:
+        fast_writer = False
```

Comment:
Am I understanding right that the fast writer is incompatible with mixins?  If so, it would be worth mentioning that in the docs just so people don't end up using it thinking it will work.  (Or if the error message actually says that specifically, it's probably fine as-is)